ext/standard/tests/file/bug28790.cache.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+Bug #28790: Add php.ini option to disable stat cache (with cache)
+--CREDITS--
+jnoll <jnoll [at] prim [dot] hu>
+Kevin Lyda <kevin [at] lyda [dot] ie>
+--FILE--
+<?php
+
+$php = '"'.getenv('TEST_PHP_EXECUTABLE').'"';
+$phpfile = getenv('TEST_PHP_EXECUTABLE');
+$impossiblefile = $phpfile.DIRECTORY_SEPARATOR.'bug28790.impossible';
```

Comment:
```suggestion
$impossiblefile = __DIR__.DIRECTORY_SEPARATOR.'bug28790.impossible';
```
Or is there some reason this is using the directory of the PHP binary?

Reply:
I'm not using the directory of the php binary, I'm using the php binary path. Say `$phpfile` is `/usr/local/bin/php`. Then `/usr/local/bin/php/bug28790.impossible` is a file that I know does not exist on every platform. So I'd rather leave this as is.

Reply:
They are both equally impossible, but the test directory is easier to reason about. TEST_PHP_EXECUTABLE could be something like `php`, not necessarily an absolute path.

Reply:
Um.  I'm not clear I understand how a relative path is harder to reason about but ok.  However `__DIR__.DIRECTORY_SEPARATOR.'bug28790.impossible` is still a possible file. Is there a variable that points to the fully qualified path for the test script being run? Essentially I need a file name to add `/bug28790.impossible` to because if `/some/file` exists, `/some/file/impossible` can't exist. Like, if I knew this was always running on a POSIX system I'd just use `/dev/null/impossible` because things would need to be very wrong for that to exist.

Reply:
Maybe you're looking for
```suggestion
$impossiblefile = __FILE__.DIRECTORY_SEPARATOR.'bug28790.impossible';
```
then?

Reply:
Done.